src/Aquifer.Jobs/Services/TranslationPostProcessingService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections.ObjectModel;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Aquifer.Jobs.Services;
+
+public interface ITranslationPostProcessingService
+{
+    public Task<string> PostProcessHtmlAsync(string html, string languageIso6393Code, CancellationToken cancellationToken);
+}
+
+public interface ILanguageSpecificTranslationPostProcessingService
+{
+    public string Iso6393Code { get; }
+    public Task<string> PostProcessHtmlAsync(string html, CancellationToken cancellationToken);
+}
+
+public static class TranslationPostProcessingServicesRegistry
+{
+    public static IServiceCollection AddTranslationPostProcessingServices(this IServiceCollection services)
+    {
+        services.AddSingleton<ITranslationPostProcessingService, TranslationPostProcessingService>();
+
+        foreach (var implementationType in AppDomain.CurrentDomain.GetAssemblies().SelectMany(s => s.GetTypes())
```

Comment:
Any new language-specific post-processing services will automatically be included without requiring any explicit DI registration due to this reflection.

I also tried out a keyed services DI strategy (.net 8+) but I like this one better.